crates/utils/src/tracing.rs

```diff
@@ -112,8 +112,10 @@ mod test {
 
     /// init tracing subscriber
     fn init() -> Result<(), Box<dyn std::error::Error>> {
-        let jaeger_online_layer = opentelemetry_jaeger::new_agent_pipeline()
-            .with_service_name("test")
+        let otlp_exporter = opentelemetry_otlp::new_exporter().tonic();
+        let jaeger_online_layer = opentelemetry_otlp::new_pipeline()
+            .tracing()
+            .with_exporter(otlp_exporter)
```

Comment:
Why is this change required? Please also rename `jaeger_online_layer`.

Reply:
The opentelemetry_jaeger::new_agent_pipeline is deprecated since 0.21.0. We'd better migrate to opentelemetry-otlp exporter. FYI: https://docs.rs/opentelemetry-jaeger/latest/opentelemetry_jaeger/fn.new_agent_pipeline.html

Reply:
I think it's no long named `jaeger`

Reply:
But it's indeed use for jaeger. The jaeger is supported otlp now.